build-settings-logic/src/main/kotlin/customization.kt

```diff
@@ -1,64 +0,0 @@
-/*
- * Copyright 2014-2024 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
- */
-
-import com.gradle.develocity.agent.gradle.DevelocityConfiguration
-import org.gradle.api.initialization.Settings
-
-fun Settings.enrichTeamCityData() {
```

Comment:
Why did you remove all this?

Reply:
> as they're [already done](https://github.com/gradle/common-custom-user-data-gradle-plugin/blob/v2.1/src/main/java/com/gradle/CustomBuildScanEnhancements.java) in common-custom-user-data-gradle-plugin.

So now we have these values duplicated in build scans:
<img width="738" alt="image" src="https://github.com/user-attachments/assets/f098e93f-b7df-4d59-b3a8-0923531005b1" />